python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -331,3 +332,71 @@ def get_units(value):
         return _unit_map[value.lower()]
 
     return value
+
+
+class Container(object):
+    def __init__(self, arg):
+        self.arg = arg
+
+
+class DateOffset(pd.DateOffset, Container):
+    def __init__(self, n=1, normalize=False, **kwds):
+        if normalize:
+            raise NotImplementedError(
+                "normalize not yet supported for DateOffset"
+            )
+
+        # TODO: Pandas supports combinations
+        if len(kwds) > 1:
+            raise NotImplementedError(
+                "only a single unit may" "be specified at a time"
+            )
```

Comment:
This message should be cleaned up to indicate that cuDF doesn't *yet* support passing multiple units

Reply:
Changed this